robot_ws/src/cloudwatch_robot/launch/await_commands.launch.py

```diff
@@ -0,0 +1,24 @@
+import os
+import sys
+
+import launch
+import launch_ros.actions
+from ament_index_python.packages import get_package_share_directory
+
+def generate_launch_description():
+    launch_actions = [
+        launch.actions.DeclareLaunchArgument(
+            name='use_sim_time',
+            default_value='true'
+        ),
+        launch.actions.IncludeLaunchDescription(
+            launch.launch_description_sources.PythonLaunchDescriptionSource(
+                [get_package_share_directory('cloudwatch_robot'), '/launch/monitoring.launch.py']
```

Comment:
os.path.join instead of `/`
(occurs in multiple places so ditto for all other as well)

Reply:
I tried doing this and it doesn't work, I think because `get_package_share_directory` doesn't resolve correctly until later, so when doing the join in this spot it resolves to an empty string.

Reply:
Upon further investigation the issue is that when using `os.path.join` if the second argument starts with / it treats it as an absolute path and ignores the first argument. I'd rather keep it with a / in the front to be consistent with how ROS1 functions (with a leading slash before launch file).

Reply:
I'm not sure I understand the reasoning. The slash signifies a path and the platform-independent way to build paths is using os.path.join.